tests/basic/storage_pool_test.go

```diff
@@ -8282,3 +8282,106 @@ var _ = Describe("{AddDiskAddDriveAndDeleteInstance}", func() {
 		AfterEachTest(contexts)
 	})
 })
+
+var _ = Describe("{DriveAddAsJournal}", func() {
+	/*
+		Add drive when as journal
+		case1:if dmthin journal is not supported so it should fail with  error message
+		case2:if it is btrfs and journal drive exists so it should have failed with error message jounral drive exists
+		case3:if it is btrfs and journal drive does not exists so it add journal drive successfully
+
+	*/
+	var testrailID = 0
+	// Testrail Description : Add drive when as journal
+	var runID int
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("DriveAddAsJournal",
+			"Add drive when as journal",
+			nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+	var contexts []*scheduler.Context
+	stepLog := "Add drive when as journal"
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("adddriveasjournal-%d", i))...)
+		}
+		ValidateApplications(contexts)
+		defer appsValidateAndDestroy(contexts)
+
+		// Get Pool with running IO on the cluster
+		poolUUID, err := GetPoolIDWithIOs(contexts)
+		log.FailOnError(err, "Failed to get pool running with IO")
+		log.InfoD("Pool UUID on which IO is running [%s]", poolUUID)
+
+		// Get Node Details of the Pool with IO
+		nodeDetail, err := GetNodeWithGivenPoolID(poolUUID)
+		log.FailOnError(err, "Failed to get Node Details from PoolUUID [%v]", poolUUID)
+		log.InfoD("Pool with UUID [%v] present in Node [%v]", poolUUID, nodeDetail.Name)
+		// Get PX StorageCluster
+		var dmthinEnabled bool
+		cluster, err := Inst().V.GetDriver()
+		if err != nil {
+			log.FailOnError(err, "Error getting clusterspecs")
+		}
```

Comment:
This doesn't need to go inside `if err != nil {}`, see how you used it above

Reply:
Done